src/content/changes/highlights/2022-01-28.md

```diff
@@ -0,0 +1,13 @@
+---
+date: 2022-01-28T10:00:00
+title: Highlights for the week ending January 28, 2022
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Apps
+
+* [external-dns-app](https://github.com/giantswarm/external-dns-app/) [v2.9.0](https://github.com/giantswarm/external-dns-app/blob/master/CHANGELOG.md#290---2022-01-24) contains changes to mitigate rate limiting on AWS clusters. Please check the changelog for additional notes if you already specify '--aws-batch-change-interval' or '--aws-zones-cache-duration'.
+- [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) [v2.8.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#280---2022-01-27) contains a potential breaking change, and disables the configuration setting 'use-forwarded-headers' by default, to increase security. Please check the changelog for additional notes if you require this configuration.
```

Comment:
```suggestion
- [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) [v2.8.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#280---2022-01-27) contains a potential breaking change, and disables the configuration setting 'use-forwarded-headers' by default, to increase security. Please check the changelog for additional notes if you require this configuration.

## Documentation

- We added documentation for [creating an app catalog](https://docs.giantswarm.io/app-platform/create-catalog/) for deploying apps using Helm charts from the community or publishing your own apps.
```

@JosephSalisbury OK if we add this from honeybadger?

cc @gianfranco-l

Reply:
@rossf7 whatever you need bud <3

Reply:
Thanks @JosephSalisbury :) I'll get this added then. As we've had this gap in docs for a while.

Reply:
@JosephSalisbury  that's all from honeybadger side :D